### PR TITLE
feat(mono): add post options for elastic search

### PR DESCRIPTION
**Pull request by LemonFace0309**
image

### CODE REVIEW

File: apps/server/src/apollo/resolvers/Query/posts.ts [hunk 1]
```diff
@@ -1,6 +1,23 @@
+import type { QueryPostsArgs } from "@src/__generated__/graphql";
 import type { Context } from "@src/server/context";
 
-export const posts = async (_root: unknown, _args: never, ctx: Context) => {
+export const posts = async (
+  _root: unknown,
+  args: QueryPostsArgs,
+  ctx: Context
+) => {
+  if (args.options && args.options.beds) {
```
Comment (chamod-gamage) [inline apps/server/src/apollo/resolvers/Query/posts.ts:9]: I think we should rename `beds` to `availableBeds` here just to reduce confusion

**Review by chamod-gamage — APPROVED** [review]
LGTM 🔥

File: apps/server/src/apollo/resolvers/Query/posts.ts (hunk 1 above)
Comment (chamod-gamage) [reply apps/server/src/apollo/resolvers/Query/posts.ts:9]: And in the other places as well

Comment (LemonFace0309) [reply apps/server/src/apollo/resolvers/Query/posts.ts:9]: on it

**Pull request merged**